src/releaseNotesProvider.ts

```diff
@@ -0,0 +1,180 @@
+import { env, ExtensionContext } from "vscode";
+import * as vscode from "vscode";
+import * as semver from "semver";
+import { Remarkable } from "remarkable";
+import http from "https";
+
+const versionKey = "metals-server-version";
+
+// prettier-ignore
+const releaseNotesMarkdownUrl: Record<string, string> = {
+  "0.11.5": "https://raw.githubusercontent.com/scalameta/metals/main/website/blog/2022-04-28-aluminium.md",
+  "0.11.6": "https://raw.githubusercontent.com/scalameta/metals/main/website/blog/2022-06-03-aluminium.md",
+};
```

Comment:
The only downside of current approach is a need for adding link to release notes here. I'll try to think of sth better, but maybe this won't be a blocker?

Reply:
Could we fail the build if the map is no updated for the default server version?